maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -425,22 +425,30 @@ class Maestro(private val driver: Driver) : AutoCloseable {
     fun findElementBySize(width: Int?, height: Int?, tolerance: Int?, timeoutMs: Long): UiElement? {
         LOGGER.info("Looking for element by size: $width x $height (tolerance $tolerance) (timeout $timeoutMs)")
 
-        return findElementWithTimeout(timeoutMs, Filters.sizeMatches(width, height, tolerance).asFilter())?.element
+        return findElementWithTimeout(
+            timeoutMs,
+            Filters.sizeMatches(width, height, tolerance).asFilter(),
+            null
+        )?.element
     }
 
     fun findElementWithTimeout(
         timeoutMs: Long,
         filter: ElementFilter,
+        viewHierarchy: ViewHierarchy?
```

Comment:
```suggestion
        viewHierarchy: ViewHierarchy? = null
```

Reply:
You can set `null` as a default value here to avoid having to pass `null` at all existing callsites

Reply:
Yes missed this. Thanks